recipes/list_items/contact_info/contact_info.vue

```diff
@@ -7,12 +7,13 @@
     class="dt-contact-info"
   >
     <template #left>
-      <div
+      <button
         v-if="showAvatar"
-        class="d-ps-relative"
+        class="d-ps-relative d-bgc-transparent d-baw0"
         data-qa="contact-info-left"
+        @click="avatarClick"
       >
```

Comment:
Going to need an aria-label that says what this button does. Probably needs to be a prop for translation reasons.